src/sentry/tasks/post_process.py

```diff
@@ -1366,8 +1374,11 @@ def detect_new_escalation(job: PostProcessJob):
         "projects:first-event-severity-new-escalation", job["event"].project
     ):
         return
-    group_age_hours = (datetime.now() - group.first_seen).total_seconds() / 3600
-    if group_age_hours >= MAX_NEW_ESCALATION_AGE_HOURS or group.substatus != GroupSubStatus.NEW:
+    group_age_hours = (timezone.now() - group.first_seen).total_seconds() / 3600
```

Comment:
i think `datetime.now()` originally worked because our test data had `first_seen` set with a `datetime`, but based on the table schema it seems like we save groups with timezone info?